src/NuGet.Core/NuGet.Commands/RestoreCommand/RequestFactory/NoOpRestoreResult.cs

```diff
@@ -16,7 +16,7 @@ namespace NuGet.Commands
     public class NoOpRestoreResult : RestoreResult
     {
         public NoOpRestoreResult(bool success, LockFile lockFile, LockFile previousLockFile, string lockFilePath, CacheFile cacheFile, string cacheFilePath, ProjectStyle projectStyle, TimeSpan elapsedTime) :
-            base(success : success, restoreGraphs : null, compatibilityCheckResults : null , 
```

Comment:
We try to do a linq operation on that value, so NPE happens (this should never happen now, since we force)